pkg/apiserver/handlers/webhook/mutation_crd_test.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhook
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	admv1 "k8s.io/api/admission/v1"
+
+	"antrea.io/antrea/pkg/controller/networkpolicy"
+)
+
+var (
+	ar = &admv1.AdmissionReview{
+		Request: &admv1.AdmissionRequest{
+			UID: "uid",
+		},
+	}
+	internalErrorAr = &admv1.AdmissionReview{
+		Request: &admv1.AdmissionRequest{
+			UID: "error uid",
+		},
+	}
+
+	requestBody, _              = json.Marshal(ar)
+	internalErrorRequestBody, _ = json.Marshal(internalErrorAr)
```

Comment:
@hjiajing please fix the golangci-lint error.

Reply:
removed the unused variable, thanks for the reminder.